Stripe/PublicHeaders/STPPaymentConfiguration.h

```diff
@@ -109,6 +109,15 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, assign, readwrite) BOOL canDeletePaymentMethods;
 
+/**
+ In order to perform API requests on behalf of a connected account, e.g. to
+ create a source on a connected account, set this property to the ID of the
+ account for which this request is being made.
+
+ @see https://stripe.com/docs/connect/authentication#authentication-via-the-stripe-account-header
```

Comment:
That docs page says this authentication option works with the platform **secret** key (I'm nearly positive you can also use the platform publishable key too: DOCS-529).

Does the publishable key work? Should we update the docs?

Reply:
Yes, the publishable key does work—this is what Stripe.js does when you set the `stripeAccount` property: https://stripe.com/docs/stripe-js/reference#stripe-function

This is somewhat covered, since tokenization can & should be done with a publishable key. It's also discussed in the sources documentation: https://stripe.com/docs/sources/connect#creating-direct-charges

Reply:
thanks @fred-stripe ! :)